apps/test/unit/lib/kits/maker/ui/SetupChecklistTest.js

```diff
@@ -19,23 +19,64 @@ describe('SetupChecklist', () => {
   const WAITING_ICON = '.fa-clock-o';
   const SUCCESS_ICON = '.fa-check-circle';
   const FAILURE_ICON = '.fa-times-circle';
+  const error = new Error('test error');
 
   beforeEach(() => {
     sinon.stub(utils, 'reload');
     sinon.stub(window.console, 'error');
-    checker = new StubSetupChecker();
+    sinon
+      .stub(SetupChecker.prototype, 'detectSupportedBrowser')
+      .callsFake(() => {
+        return Promise.resolve();
+      });
```

Comment:
super nit: we can slightly simplify these fakes, which i think will make this more readable

```javascript
sinon
      .stub(SetupChecker.prototype, 'detectSupportedBrowser')
      .callsFake(() => Promise.resolve());

// below should also work but i think this syntax confuses people sometimes
sinon
      .stub(SetupChecker.prototype, 'detectSupportedBrowser')
      .callsFake(Promise.resolve);
```